assets/js/app/Data.js

```diff
@@ -112,6 +185,7 @@ LocusZoom.Data.LDSource = function(url) {
         }
     };
 
+    var that = this;
```

Comment:
[See other note regarding this/that/bind](https://github.com/statgen/locuszoom/pull/7/files#r48358203)